src/routes/identity/account/+page.svelte

```diff
@@ -27,6 +28,28 @@
 		copy(address)
 		copied = true
 	}
+
+	async function getBalances() {
+		const address = $walletStore.wallet?.address
+		if (address === undefined) return
+
+		const balance = await getBalance(address)
+		console.debug({ balance, $balanceStore })
+
+		balanceStore.update((balanceState) => ({
+			...balanceState,
+			balances: balanceState.balances.map((value) => {
+				if (value.address) {
```

Comment:
Currently the `getBalance` only returns the balance of the native token. The `value.address` is the contract address of the token and it is `undefined` when it's the native token. Therefore this code just fetches the native token balance and updates it, while keeping the other tokens unchanged.

In order to make the non-native tokens to work, there needs to be added a function that fetches the token balance from the contract using the address. This is currently missing and this could be done as a separate issue. I will open one.

Reply:
https://github.com/logos-innovation-lab/waku-objects-playground/issues/168